benchmarks/child-creation.bench.js

```diff
@@ -60,6 +60,13 @@ const run = bench([
       child.info({ hello: 'world' })
     }
     setImmediate(cb)
+  },
+  function benchPinoCreationWithOption (cb) {
+    const child = plogDest.child({ a: 'property' }, { redact: [] })
+    for (var i = 0; i < max; i++) {
```

Comment:
why not let?

Reply:
the looping is copy from other test. As I remember, `for-var` loop out perform `for-let` loop. I think that is the reason.

Reply:
huh, interesting. TIL